### PR TITLE
Patch Dynamic DNS Hover.com

### PR DESCRIPTION
**Pull request by xygrec**
Changes needed to support dynamic dns service for Hover.com

Tested with 6 hosts on 4 different domains.

### CODE REVIEW

**Comment by pfSense-git-updates** [comment]
Before this pull request can be accepted you must first sign a CLA as described at https://www.pfsense.org/about-pfsense/#cla. Please read for more details.

**Comment by xygrec** [comment]
Signed the CLA.

**Comment by xygrec** [comment]
Any timeline for this change?

File: src/etc/inc/services.inc [hunk 1]
```diff
@@ -23,8 +23,8 @@
...
+define('DYNDNS_PROVIDER_VALUES', 'all-inkl citynetwork cloudflare cloudflare-v6 custom custom-v6 dnsexit dnsimple dnsmadeeasy dnsomatic duiadns duiadns-v6 dyndns dyndns-custom dyndns-static dyns easydns eurodns freedns freedns-v6 glesys googledomains gratisdns he-net he-net-v6 he-net-tunnelbroker loopia namecheap noip noip-free ods opendns ovh-dynhost route53 selfhost spdyn spdyn-v6 zoneedit hover');
+define('DYNDNS_PROVIDER_DESCRIPTIONS', 'All-Inkl.com,City Network,CloudFlare,CloudFlare (v6),Custom,Custom (v6),DNSexit,DNSimple,DNS Made Easy,DNS-O-Matic,DuiaDns.net,DuiaDns.net (v6),DynDNS (dynamic),DynDNS (custom),DynDNS (static),DyNS,easyDNS,Euro Dns,freeDNS,freeDNS (v6),GleSYS,Google Domains,GratisDNS,HE.net,HE.net (v6),HE.net Tunnelbroker,Loopia,Namecheap,No-IP,No-IP (free),ODS.org,OpenDNS,OVH DynHOST,Route 53,SelfHost,SPDYN,SPDYN (v6),ZoneEdit,Hover');
```
Comment (rbgarga) [inline src/etc/inc/services.inc]: Please keep this list alphabetically sorted

**Comment by xygrec** [comment]
Done

**Comment by pedigree** [comment]
Is there any reason why hover SSL verification is disabled?

**Comment by xygrec** [comment]
Good catch.
Adjusted with pull request #3723.

**Comment by nickpdawson** [comment]
just curious when we might see this patch deployed? In latest 2.4 as of August 27, 2017 Hover still isn't an option.

**Comment by jim-p** [comment]
@nickpdawson Yes it is. It's between HE.net and Loopia on both 2.4 and 2.3.4.

**Comment by nickpdawson** [comment]
@jim-p I am so sorry! I totally mis-read. I found this issue when looking for ways to use hover for DNS with the ACME package. I'm very pleased, though, to see that Hover is supported in dyndns. So while it wasn't my original challenge, it's nonetheless a welcome surprise. Sorry for the mis-led question on my part!

**Pull request merged**